app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Product;
+use App\Comment;
+use RealRashid\SweetAlert\Facades\Alert;
+
+
+class CommentController extends Controller
+{
+    public function createComment(Request $request, $id)
+    {
+        $data = $request->all();
+        $data['user_id'] = auth()->id();
+
+        if (auth()->check()) {
+            $username = auth()->user()->name;
+        }
+
+        $images = Product::findOrFail($id)->images;
+        if (isset($images->first()->link_to_image)) {
```

Comment:
```suggestion
        if (isset($images->first())) {
```
như cũ là lỗi luôn á